.github/workflows/renovate-automatic-approval.yml

```diff
@@ -6,18 +6,19 @@ on:
 jobs:
   approve-renovate-prs:
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout code
```

Comment:
Checkout no longer needed?

Reply:
What would we need the checkout for anyway? This uses GH CLI and nothing else

Reply:
I thought it would be required that gh knows which repository it should get the pr's from